plugin_aliases.yml

```
@@ -0,0 +1,2 @@
+input:
+  beats: elastic_agent
```

I personally read aliases the opposite way, and would expect the above to indicate that the `beats` alias is pointing to the `elastic_agent` plugin. The structure as-above also allows us to define an alias multiple times (which would be non-deterministic) and prevents us from using multiple aliases toward a single plugin (which we may or may not want).

```suggestion
  elastic_agent: beats
```

---

Sounds good!

---

done